android/engine/src/main/java/org/smartregister/fhircore/engine/task/FhirCarePlanGenerator.kt

```diff
@@ -209,7 +228,15 @@ constructor(
           definition.dynamicValue.forEach { dynamicValue ->
             if (definition.kind == ActivityDefinition.ActivityDefinitionKind.CAREPLAN) {
               dynamicValue.expression.expression
-                .let { fhirPathEngine.evaluate(null, input, planDefinition, subject, it) }
+                .let {
+                  fhirPathEngine.evaluate(
+                    null,
+                    input,
+                    planDefinition,
+                    subject,
+                    it,
```

Comment:
You can use named parameters to improve code readablity.

Reply:
change in code were not introduced by my commit and were there from before

Reply:
Done https://github.com/opensrp/fhircore/issues/3386